README.md

```diff
@@ -2,6 +2,21 @@
 
 Ruby client for the [kubernetes](http://kubernetes.io/) API.
 
+## Usage
+```ruby
+require 'kubernetes'
+require 'pp'
+
+kube_config = Kubernetes::KubeConfig.new("#{ENV['HOME']}/.kube/config")
+config = Kubernetes::Configuration.new()
+
+kube_config.configure(config)
+
```

Comment:
So this modifies the `config` object based on the kube_config content?
This is unintuitive to read in Ruby, consider adding an explanatory comment.